src/Minsk.Generators/SyntaxNodeGetChildrenGenerator.cs

```diff
@@ -23,40 +23,42 @@ public void Execute(SourceGeneratorContext context)
 #pragma warning disable IDE0063 // Use simple 'using' statement: we want to control when the 'using' varibales go out of scope
             SourceText sourceText;
 
-            var compilation = (CSharpCompilation)context.Compilation;
+            CSharpCompilation? compilation = (CSharpCompilation)context.Compilation;
```

Comment:
The fix all for using var seems to be buggy!
I have `csharp_style_var_when_type_is_apparent = true` but the code fix changed var to the type.

Probably the analyzer *thinks* that the actual type `CSharpCompilation?` isn't apparent from the right side `CSharpCompilation`.